### DISCUSSION TITLE
OG-Core Developer Meeting, Apr. 14, noon-1pm ET

### DISCUSSION THREAD

**Discussion by rickecon** [PSLmodels/OG-Core#866] [Meeting]
**Agenda for OG-Core developer meeting: 4/24/23**
12:00-1:00pm ET

Join Google Meet video conference
https://meet.google.com/ynf-zpdr-pbx

Join by phone
‪(US) +1 252-394-8749‬ PIN: ‪321 186 757‬#

This meeting is open to the public.

**Agenda**
* MERGED: 2D smoothing spline in OG-Core, PR #839 by @prrathi 
* OG-USA [PR #69](https://github.com/PSLmodels/OG-USA/pull/69) test failures

Next meeting: Monday, May 8, noon-1pm ET

@rickecon @jdebacker @jpycroft @nikhilwoodruff @SeaCelo @sarvonz @prrathi

**Comment by jpycroft** [comment]
Hi Rick,
Sorry, I'll not be around this evening.
Let me know if I can help with anything.
Best,
Jon

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: Richard Evans ***@***.***>
Sent: Monday, April 24, 2023 8:26:05 AM
To: PSLmodels/OG-Core ***@***.***>
Cc: jpycroft ***@***.***>; Mention ***@***.***>
Subject: [PSLmodels/OG-Core] OG-Core Developer Meeting, Apr. 14, noon-1pm ET (Discussion #866)


Agenda for OG-Core developer meeting: 4/24/23
12:00-1:00pm ET

Join Google Meet video conference
https://meet.google.com/ynf-zpdr-pbx

Join by phone
‪(US) +1 252-394-8749‬ PIN: ‪321 186 757‬#

This meeting is open to the public.

Agenda

  *   MERGED: 2D smoothing spline in OG-Core, PR #839<https://github.com/PSLmodels/OG-Core/pull/839> by @prrathi<https://github.com/prrathi>
  *   OG-USA PR #69<https://github.com/PSLmodels/OG-USA/pull/69> test failures

Next meeting: Monday, May 8, noon-1pm ET

@rickecon<https://github.com/rickecon> @jdebacker<https://github.com/jdebacker> @jpycroft<https://github.com/jpycroft> @nikhilwoodruff<https://github.com/nikhilwoodruff> @SeaCelo<https://github.com/SeaCelo> @sarvonz<https://github.com/sarvonz> @prrathi<https://github.com/prrathi>
